package.json

```diff
@@ -1,19 +1,48 @@
 {
   "name": "holycraap-front",
   "version": "0.1.0",
+  "description": "In search of the one and only Holy Craap - a treasure also known as \"Big Whoop(TM)\" - a new adventure game written in Java and JavaScript, using Spring and React.",
   "private": true,
   "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "@types/jest": "^24.9.1",
+    "@types/node": "^12.12.47",
+    "@types/react": "^16.9.41",
+    "@types/react-dom": "^16.9.8",
+    "npm-watch": "^0.6.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "^3.4.1"
+    "react-scripts": "3.4.1",
+    "typescript": "^3.7.5"
+  },
+  "watch": {
+    "test": "{src,test}/*.js"
   },
-  "devDependencies": {},
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "test": "tape test/*.js",
```

Comment:
`jest` that was used before works with TypeScript

Reply:
should be better now